department-of-veterans-affairs/va.gov-cms
# VACMS-4685: Vet-Center-Page-Intro-Default

…ent type

## Description

See #4685

## Testing done


## Screenshots


## QA steps

- Login to PR environment

- Create a new Vet Center node: `/node/add/vet_center`

- [ ] Confirm that the Page Introduction is pre-populated with the following:
`We offer confidential help for Veterans, service members, and their families at no cost in a non-medical setting. Our services include counseling for needs such as depression, posttraumatic stress disorder (PTSD), and the psychological effects of military sexual trauma (MST). We can also connect you with more support in VA and your community.
`

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
